packages/arb-token-bridge-ui/src/components/common/Header.tsx

```diff
@@ -150,6 +150,45 @@ const LoginButton: React.FC = () => {
   )
 }
 
+const AddNetworkButton: React.FC = () => {
+  const {
+    app: { networkID, changeNetwork, l2NetworkDetails }
+  } = useAppState()
+
+  const hide = useMemo(() => {
+    return (
+      !networkID ||
+      !l2NetworkDetails ||
+      !changeNetwork ||
+      (l2NetworkDetails && l2NetworkDetails.chainID === networkID)
+    )
+  }, [networkID, l2NetworkDetails])
+
+  return (
+    <>
+      {hide ? null : (
+        <button
+          onClick={() => {
+            const chainID: string | null =
+              l2NetworkDetails && l2NetworkDetails.chainID
+
+            if (!chainID || !changeNetwork) {
+              console.log("Can't add L2 network")
+              return
+            }
+            changeNetwork(chainID)
+          }}
+          type="button"
+          className="mr-4 text-white hover:text-navy hover:text-gray-200 hover:bg-gray-200 cursor-pointer z-50 rounded-md text-sm font-medium"
+          style={{ padding: '10px 12px' }}
+        >
+          Add L2 Network
```

Comment:
We could instead say `Switch to L2 Network`.

Reply:
"adding" (from not having the network included at all in metamask) is the more important action we want to convey (for, i.e., a user that doesn't necessarily want to bridge, but just add arb to metamask so they can go flip NFTs). 
"switching" we kind don't want the users thinking about here, since we handle that automatically jit, imo